pizzakit-plugin/includes/pizzakit.php

```diff
@@ -126,12 +134,46 @@ public static function swish_callback_handler($_data)
 						if(!$res){
 							trigger_error("Pizzakit: error creating entry in orders");
 						}
+						$query = $wpdb->prepare('SELECT id,location,name,email from '.$table.' WHERE id = %d',$order_id);
+						$res = $wpdb->get_results($query);
+						trigger_error(print_r($res,$return=true));
+						Pizzakit::send_payment_confirmation($res[0]);
 					}
 				}
 			}
 		}
 	}
 
+	private static function send_payment_confirmation($details){
+		$mail = new PHPMailer(true);
+		try {
+			//Server settings
+			$mail->SMTPDebug = SMTP::DEBUG_SERVER;                      // Enable verbose debug output
+			$mail->isSMTP();                                            // Send using SMTP
+			$mail->Host       = 'smtp.server.example';                    // Set the SMTP server to send through
+			$mail->SMTPAuth   = true;                                   // Enable SMTP authentication
+			$mail->Username   = 'admin';                     // SMTP username
+			$mail->Password   = 'hunter2';                               // SMTP password
+			$mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;         // Enable TLS encryption; `PHPMailer::ENCRYPTION_SMTPS` encouraged
+			$mail->Port       = 587;                                    // TCP port to connect to, use 465 for `PHPMailer::ENCRYPTION_SMTPS` above
+		
+			//Recipients
+			$mail->setFrom('from addr@coolsvile.cs', 'Name');
+			$mail->addAddress($details->email, 'Customer');     // Add a recipient
+			$mail->addReplyTo('from addr@coolsvile.cs', 'Information');
+		
+			// Content
+			$mail->isHTML(true);                                  // Set email format to HTML
+			$mail->Subject = "Menomale pizza kit order ".$details->id;
+			$mail->Body    = "<h3>Hej, vi har tagit emot din order</h3><p>Order ".$details->id." registrerad under namn".$details->name."</p><p>hämta upp din order från fredag - söndag denna vecka på Menomale i ".$details->location."</p>";
```

Comment:
Can you use the settings for pickup days to generate the days?  
`get_site_option('pizzakit_time_pickup_start_day')`  
`get_site_option('pizzakit_time_pickup_end_day')`

And add spaces before the name.